content/en/load-testing/Manual/320-test-evaluation.md

```diff
@@ -195,7 +195,7 @@ When you started the master controller in [interactive mode](../310-test-executi
 
 The master controller's command line menu in interactive mode looks like this:
 
-```bash
+```dos
```

Comment:
DOS? Not bash?

Reply:
bash is giving us line numbers and code highlighting. kinda unneccessary and not what you get in real-life console output.